src/util/device.rs

```diff
@@ -1,5 +1,7 @@
 use std::convert::TryFrom;
 
+use crate::COPY_BUFFER_ALIGNMENT;
```

Comment:
let's follow the existing style please, where `crate::COPY_BUFFER_ALIGNMENT` is fully qualified where it's useed